art/attacks/evasion/auto_projected_gradient_descent.py

```diff
@@ -246,7 +179,14 @@ def __call__(self, y_true, y_pred):
 
```

Comment:
Please change line 149 from `class DifferenceLogitsRatioTensorFlowV2:` to `class DifferenceLogitsRatioTensorFlowV2(TensorFlowV2Loss):`